samples/multiplatform/greeting/android/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'kotlin-platform-jvm'
+
+// Specify a group and a version of the library to access it in Android Studio.
+// By default the project directory name is used as an artifact name thus the full dependency
+// description will be 'org.greeting:android:1.0'
+group = 'org.greeting'
+version = 1.0
```

Comment:
maven-style dependency is actually not used in this sample anymore, only project dependency

Reply:
We will use the maven-style dependency in a separate repo.